pyartcd/pyartcd/pipelines/gen_assembly.py

```diff
@@ -129,27 +130,14 @@ async def run(self):
             await self._slack_client.say(f"Error generating assembly definition for {self.assembly}", slack_thread)
             raise
 
-    async def _get_latest_accepted_nightly(self):
+    def _get_latest_accepted_nightly(self):
```

Comment:
Let's not change this async function to sync.

Reply:
updated in https://github.com/openshift-eng/art-tools/pull/1097/commits/b90c644869b508bc909274fb7c7c1f20e2f6c894